src/H5VLint.c

```diff
@@ -2623,3 +2621,42 @@ H5VL_setup_idx_args(hid_t loc_id, const char *name, H5_index_t idx_type, H5_iter
     FUNC_LEAVE_NOAPI(ret_value)
 } /* end H5VL_setup_idx_args() */
 
+/*-------------------------------------------------------------------------
+ * Function:    H5VL_setup_token_args
+ *
+ * Purpose:     Set up arguments to access an object by token
+ *
+ * Return:      SUCCEED / FAIL
+ *
+ *-------------------------------------------------------------------------
+ */
+herr_t
+H5VL_setup_token_args(hid_t loc_id, H5R_ref_t *ref_ptr, H5O_token_t obj_token,
+                      H5VL_object_t **vol_obj, H5VL_loc_params_t *loc_params)
+{
+    herr_t      ret_value = SUCCEED; /* Return value */
+
+    FUNC_ENTER_NOAPI(FAIL)
+
+    /* Sanity check */
+    HDassert(vol_obj);
+    HDassert(loc_params);
+
+    /* Get object token */
+ /* This caused the need to include H5Rpkg.h, which is not allowed.  Right?
```

Comment:
Yeah, I'd be pretty reluctant to include H5Rpkg.h in this file.   Is there some other way to structure this routine's arguments?